### DISCUSSION TITLE
单独批量下载视频的字幕/弹幕

### DISCUSSION THREAD

**Discussion by ZzzzRyan** [the1812/Bilibili-Evolved#4581] [功能建议 / Ideas]
## 🙌 功能建议：
希望单独支持批量下载字幕和批量下载弹幕。

## 💡 功能用途：
部分系列老视频已完成下载，但想下载视频新的弹幕、以及为曾经没有下载字幕的视频下载字幕。

**Comment by brookechuo** [comment]
顶一下。
同样需要批量下载字幕。
恳请作者考虑添加这个功能。
非常感谢！

**Reply by Geeyun-JY3** [reply]
看置顶：#1297，根据这个讨论被打的 label 并善用搜索，作者已经实现该功能了：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.8
或许可以参考：#5261
